README.md

```diff
@@ -244,6 +245,38 @@ Below is a short synopsis of the CLI commands. For detailed options, specify the
 
 Performs one-time initialization of EveryAuth on a developer machine. This command will create a free Fusebit account and store the credentials necessary to access it in your home directory's `~/.fusebit/settings.json` file. Keep this file secret. You can also move the `.fusebit` directory to a new machine from which you want to access your EveryAuth configuration, like a CI/CD box or a second development machine. 
 
+#### everyauth profile export
+
+Exports to `stdout` a JSON-encoded profile object which can be used with `everyauth profile import`, or
+set in the environment after base64 encoding within `EVERYAUTH_PROFILE_JSON` to support generating keys in
+production to authenticate to the EveryAuth backend.
+
+**Example:** Encode the current profile to generate JWT keys dynamically in production, and store it in a
+`.env` file.
+
+```
+echo EVERYAUTH_PROFILE_JSON=`everyauth profile export | base64` >> .env
+```
+
+#### everyauth profile import
+
+Supports importing, from `stdin` or a file, a previously existing profile.
```

Comment:
Redaction: It supports importing a previously existing profile from `stdin` or a file.